packages/nimble-components/src/rich-text/components/mention-view/mention-users-view/styles.ts

```diff
@@ -0,0 +1,19 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+    mentionFont,
+    mentionFontColor
+} from '../../../../theme-provider/design-tokens';
+
+export const styles = css`
+    ${display('inline-block')}
+
+    :host {
+        box-sizing: border-box;
+        font: ${mentionFont};
+    }
+
+    .control {
+        color: ${mentionFontColor};
+    }
```

Comment:
Should we also add styling for the disabled state? (`mentionDisabledFontColor`)

Reply:
Added styling for disabled state and included in the theme matrix test too.